packages/components/tests/dummy/app/templates/components/reveal.hbs

```diff
@@ -0,0 +1,114 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Reveal Component"}}
+
+<Shw::Text::H1>Reveal</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Text::H3>On its own</Shw::Text::H3>
+
+  <Shw::Flex {{style gap="5rem"}} as |SF|>
+    <SF.Item @label="With plain text content">
+      <Hds::Reveal @text="More options">
+        Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the
+        holistic world view of disruptive innovation via workplace diversity and empowerment.
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With HTML formatted content">
+      <Hds::Reveal @text="More options">
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content">
+      <Hds::Reveal @text="More options">
+        <p>More content</p>
+        <p>See here for more info:
+          <a href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">HashiCorp</a></p>
+      </Hds::Reveal>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Used next to text</Shw::Text::H3>
+
+  <Shw::Flex @direction="column" {{style gap="2rem"}} as |SF|>
+    <SF.Item @label="Used inline with text (edge case)">
+      <p class="hds-typography-body-300">
+        Lorem ipsum dolor sit amet consectetur
+        <Hds::Reveal @text="More options">
+          Additional content
+        </Hds::Reveal>
+        adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+      </p>
+    </SF.Item>
```

Comment:
thought: the fact that this component can be used inline (at least at this point, considering the `inline-block` property) makes this example very much relevant, as this is how a consumer would see it rendered in their application next to any other inline element. Moving to a `block` element would make this obsolete, indeed.